builtin/game/item_entity.lua

```diff
@@ -131,6 +136,24 @@ core.register_entity(":__builtin:item", {
 		return true
 	end,
 
+	enable_physics = function(self)
+		if self.physical_state == false then
```

Comment:
Why do you compare to `false`?
This might cause problems with old items that have `physical_state = nil`.

Reply:
`physical_state` is temporary. It's assumed to be `true` when loading the item. Bug?

Reply:
Oh, right, the luaentity "attributes" aren't saved permanently.
Anyway, [the code style guidelines](https://dev.minetest.net/Lua_code_style_guidelines#Misc) say:
> Don't compare values explicitly to `true`, `false`, or `nil`, unless it's really needed.

Reply:
> Bug?

intentional